### PR TITLE
AddTemplate cisco_nxos_show_nve_vni_interface_nve_1_detail.textfsm

### PR DESCRIPTION
**Pull request by jpvrenen**
AddTemplate cisco_nxos_show_nve_vni_interface_nve_1_detail.textfsm

### CODE REVIEW

**Comment by ryanmerolle** [comment]
What is required to push this through. I'd be happy to help

**Comment by jpvrenen** [comment]
@itdependsnetworks @jmcgill298 @jvanderaa any chance one of you can take a look at this PR?

**Comment by jmcgill298** [comment]
@jpvrenen please fix to pass the test cases, and also, we have updated many templates to standardize on naming and have moved away from abbreviations, please change mcast_addr to multicast_address. Thanks.

**Comment by jpvrenen** [comment]
> @jpvrenen please fix to pass the test cases, and also, we have updated many templates to standardize on naming and have moved away from abbreviations, please change mcast_addr to multicast_address. Thanks.

@jmcgill298 Thanks for reviewing, I changed the abbreviations and used the cli.py to autogenerate the yaml file. I also ran the tests locally and 'All tests have passed!'.   Thanks!

File: ntc_templates/templates/index [hunk 1]
```diff
@@ -401,6 +401,7 @@ cisco_nxos_show_hostname.textfsm, .*, cisco_nxos, sh[[ow]] hostn[[ame]]
 cisco_nxos_show_hsrp_all.textfsm, .*, cisco_nxos, sh[[ow]] hsrp all
 cisco_nxos_show_ip_route.textfsm, .*, cisco_nxos, sh[[ow]] ip route
 cisco_nxos_show_feature.textfsm, .*, cisco_nxos, sh[[ow]] feat[[ure]]
+cisco_nxos_show_nve_vni_interface_nve_1_detail.textfsm, .*, cisco_nxos, sh[[ow]] nv[[e]] vn[[i]] int[[erface]] nv[[e]] 1 det[[ail]]
```
Comment (jmcgill298) [inline ntc_templates/templates/index:404]: This does not comply with the index ordering requirements: https://ntc-templates.readthedocs.io/en/latest/dev/dev_parser/#index-file

Comment (jpvrenen) [reply ntc_templates/templates/index]: @jmcgill298 Ordering has been corrected, thanks!

**Pull request merged**